api/src/resolvers/campaigns/campaigns-cypher.ts

```diff
@@ -2,7 +2,11 @@ export const SetEquipmentDeadline = `
 MATCH (gatheringService:GatheringService {id: $id})
 SET gatheringService.equipmentStartDate = date($startDate),
     gatheringService.equipmentEndDate = date($endDate)
-MERGE (equipmentDate:TimeGraph:EquipmentDate {date:date($startDate)})
+MERGE (equipmentDate:TimeGraph {date:date($startDate)})
+ON MATCH 
+SET equipmentDate:EquipmentDate
+ON CREATE
+SET equipmentDate:EquipmentDate
```

Comment:
You do realise that to have a condition, and then to run the exact same code in both cases, is tautology right? 

You should remove this ON MATCH and ON CREATE 

I don't think they are solving any problem

Reply:
Seen. Changing it